package.json

```diff
@@ -116,6 +116,7 @@
     "postcss-cssnext": "^2.7.0",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
+    "sweetalert": "^1.1.3",
```

Comment:
Oh, no, because the config reads it from `node_modules`.

Hmm—for all other libraries that get vendored in to the preview environment, we use Bower, since Bower packages are explicitly intended to run in a non-module environment, whereas with npm packages there’s only a convention of including a packaged up version that exports something to `window`. All things being equal, I’m inclined to do the same here. What do you think?